tests/unit/dataactvalidator/test_d41_detached_award_financial_assistance_3.py

```diff
@@ -13,7 +13,8 @@ def test_column_headers(database):
 
 def test_success(database):
     """ When provided, PrimaryPlaceofPerformanceZIP+4 must be in the state specified by PrimaryPlaceOfPerformanceCode.
-        Ignore cases where zip4 isn't provided """
+        In this specific submission row, the ZIP5 (and by extension the full ZIP+4) is not a valid ZIP code in the
+        state in question."""
 
     # XX00000 validates here because it passes as long as the zip is valid in that state, this is checked
```

Comment:
This comment is probably not needed. We aren't checking anything with this format. (I know it was already here, but we might as well remove it)